src/qibo/models/encodings.py

```diff
@@ -427,20 +427,23 @@ def entangling_layer(
     closed_boundary: bool = False,
     **kwargs,
 ):
-    """Create a layer of two-qubit, entangling gates.
+    """Create a layer of two-qubit entangling gates.
 
     If the chosen gate is a parametrized gate, all phases are set to :math:`0.0`.
 
     Args:
         nqubits (int): Total number of qubits in the circuit.
         architecture (str, optional): Architecture of the entangling layer.
-            Options are ``diagonal``, ``shifted``, ``even-layer``, and ``odd-layer``.
-            Defaults to ``"diagonal"``.
+            In alphabetical order, options are ``"diagonal"``, ``"even_layer"``,
+            ``"next_nearest"``, ``"odd_layer"``, ``"pyramid"``, ``"shifted"``,
+            ``"v"``, and ``"x"``. The ``"x"`` architecture is only defined for an even number
+            of qubits. Defaults to ``"diagonal"``.
         entangling_gate (str or :class:`qibo.gates.Gate`, optional): Two-qubit gate to be used
             in the entangling layer. If ``entangling_gate`` is a parametrized gate,
             all phases are initialized as :math:`0.0`. Defaults to  ``"CNOT"``.
-        closed_boundary (bool, optional): If ``True`` adds a closed-boundary condition
-            to the entangling layer. Defaults to ``False``.
+        closed_boundary (bool, optional): If ``True`` and ``architecture not in
+            ["pyramid", "v", "x"]``, adds a closed-boundary condition to the entangling layer.
+            Defaults to ``False``.
```

Comment:
Why is the default to `False`?  (curiosity)

Reply:
No particular reason